RELEASE-NOTES.md

```diff
@@ -79,7 +135,7 @@
 ### Layout
 *  Updated docs [(#1885](https://github.com/patternfly/patternfly-next/pull/1885))
 
-# PatternFly RC2 (2019-05-28)
+## 2019.01 - RC2 - release notes (2019-05-28)
```

Comment:
Can you remove the first dash to be consistent with "2019.02 RC2.1 release notes"?